restApi/src/main/java/bisq/restApi/dto/OfferDto.java

```diff
@@ -0,0 +1,40 @@
+package bisq.restApi.dto;
+
+import bisq.account.protocol.SwapProtocolType;
+import bisq.offer.Offer;
+import bisq.offer.spec.Direction;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+
+@Slf4j
+@Data
+@ToString
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
+@Schema(name = "Offer")
+public class OfferDto extends BaseDto<OfferDto> {
+
+    @EqualsAndHashCode.Include
+    protected String id;
+    protected long date; //long ???
```

Comment:
We use the value in case of wrapper types -> date as long value in dto/protobuf context, only in java domain we use Date object. Similar with Coin where we use the satoshi value as long and Coin only in higher level APIs.